demo/opae-nlb-demo/Dockerfile

```diff
@@ -13,7 +13,7 @@ FROM ${CLEAR_LINUX_BASE} as builder
 ARG CLEAR_LINUX_VERSION=
 
 RUN swupd update --no-boot-update ${CLEAR_LINUX_VERSION} && \
-    swupd bundle-add wget c-basic devpkg-json-c devpkg-util-linux devpkg-hwloc doxygen Sphinx patch
+    swupd bundle-add wget c-basic devpkg-json-c devpkg-util-linux devpkg-hwloc doxygen patch python-basic
```

Comment:
This may work today but is going fail any day. See: https://github.com/clearlinux/clr-bundles/commit/3e41ea2c2f42910fd4b4643f2e49a6114eb1e2ba

Also: https://github.com/clearlinux/clr-bundles/commit/f711703c86e95a1f3c8430461efb1faef3cebafa

Reply:
thanks. removed python-basic